doc/classes/KinematicBody.xml

```diff
@@ -32,7 +32,7 @@
 			<argument index="0" name="slide_idx" type="int">
 			</argument>
 			<description>
-				Returns a [KinematicCollision], which contains information about a collision that occured during the last [method move_and_slide] call. Since the body can collide several times in a single call to [method move_and_slide], you must specify the index of the collision in the range 0 to ([method get_slide_count]()-1).
+				Returns a [KinematicCollision], which contains information about a collision that occured during the last [method move_and_slide] call. Since the body can collide several times in a single call to [method move_and_slide], you must specify the index of the collision in the range 1 to ([method get_slide_count] - 1).
```

Comment:
That one could also be fixed as commented above.

Reply:
Actually not fully as `escape_post` works only for alphanumeric stuff and won't work with `()`. Maybe it should be loosened to escape everything but the actual stuff that don't need to be, if any (`. , : ; ? !` or so?).